README.md

```diff
@@ -55,14 +55,14 @@ This CLI creates a simple hello-world application. There is also a large readme
 
 
 # React to Preact
-<img width="699" alt="screen shot 2017-08-25 at 2 38 20 pm" src="https://user-images.githubusercontent.com/1016365/29733688-3a889c1c-89a3-11e7-86e0-2a73ce8c8c50.png">
+<img width="699" alt="React to Preact" src="https://user-images.githubusercontent.com/1016365/29733688-3a889c1c-89a3-11e7-86e0-2a73ce8c8c50.png">
```

Comment:
these don't matter because we're placing this in contentful

Reply:
word.
oh well doesnt hurt